samples/ccr/start.sh

```diff
@@ -25,7 +25,7 @@ printf "Waiting for clusters to get ready "
 ALL_CLUSTERS_READY=false
 
 while ! $ALL_CLUSTERS_READY; do
-    (curl -ks -u admin:admin https://localhost:9200 -o /dev/null && curl -ks -u admin:admin https://localhost:9201 -o /dev/null && ALL_CLUSTERS_READY=true) || (printf "." && sleep 5)
+    (curl -ks -u admin:myStrongPassword123! https://localhost:9200 -o /dev/null && curl -ks -u admin:admin https://localhost:9201 -o /dev/null && ALL_CLUSTERS_READY=true) || (printf "." && sleep 5)
```

Comment:
Not sure where the second cluster is coming from, I think the first cluster is coming from `docker-compose-metricstore`, which is on the latest release. This change will fail CI until 2.12.0 docker image is released, at which point it would fix the CI.

Reply:
Can maintainers help out here in understanding if the second cluster needs similar changes/where those that cluster's password should be set?

Reply:
I do not believe this script is being used anymore. @gkamat can you confirm?

Reply:
@IanHoang I will remove this file then, if its not being used. @gkamat can you also confirm here?